docs/notifications/deprecations/migration-tool/index.md

```
@@ -4,104 +4,620 @@
+          nickfedor/watchtower \
+          notify-upgrade \
+          --notifications gotify \
+          --notification-gotify-url "https://my.gotify.tld/" \
+          --notification-gotify-token "SuperSecretToken" \
+          --notification-gotify-tls-skip-verify
+        ```
 
-3. Replace the deprecated configuration with the coverted Shoutrrr URL:
+=== "Slack"
 
     === "Docker Compose"
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)